### DISCUSSION TITLE
Vanced Manager 2.6.1

### DISCUSSION THREAD

**Discussion by KevinX8** [TeamVanced/VancedManager#696] [General]
- Added Android 12 support
- Added a confirmation dialog for uninstalling
- Fixed a bug with the save button going offscreen when the versions list was too large
- Removed all Crowdin libraries (didn't function correctly)
- Small optimizations

<hr /><em>This discussion was created from the release <a href='https://github.com/YTVanced/VancedManager/releases/tag/v2.6.1'>Vanced Manager 2.6.1</a>.</em>

**Comment by Sappurit** [comment]
Please push an update in latest.json

**Comment by Retr0680** [comment]
Make support for android TV

**Comment by cuynu** [comment]
Update YouTube Vanced to latest version pleasee
Latest version now : 16.39.36
Current YTVanced Version : 16.29.39
